Source/Core/AudioCommon/CubebUtils.cpp

```diff
@@ -16,6 +16,45 @@
 
 static ptrdiff_t s_path_cutoff_point = 0;
 
+// Due to the way Dolphin is, we can't guarantee a cubeb context will be created and destroyed
+// from the same thread, this means that we can call CoInitialize on creation but not on destroy,
+// as we don't want to CoUninitialize a random thread, so before calling DestroyContext(),
+// we need to call CoInitialize ourselves. In case it was the same thread as the one it was created
+// on, the new init will just stack and a ScopedThreadAccess will be popped immediately,
+// if it's another thread, then the original ScopedThreadAccess will be called when the thread is
+// closed, avoiding memory leaks
+thread_local std::vector<CubebUtils::ScopedThreadAccess> scoped_thread_accesses;
```

Comment:
```suggestion
static thread_local std::vector<CubebUtils::ScopedThreadAccess> scoped_thread_accesses;
```

Seems to have been missed when moving the code

Reply:
Fixed, I had forgotten